simapp/app.go

```diff
@@ -30,6 +30,8 @@ import (
 	transfer "github.com/cosmos/cosmos-sdk/x/ibc-transfer"
 	ibcclient "github.com/cosmos/cosmos-sdk/x/ibc/02-client"
 	port "github.com/cosmos/cosmos-sdk/x/ibc/05-port"
+	ibc24host "github.com/cosmos/cosmos-sdk/x/ibc/24-host"
```

Comment:
`ibc24host` can just be `host` unless it conflicts with something (which I don't think it should) then it would be `ibchost`